modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/PostalAddressResourceImpl.java

```diff
@@ -61,6 +64,25 @@ public void deletePostalAddress(Long postalAddressId) throws Exception {
 		_addressService.deleteAddress(postalAddressId);
 	}
 
+	@Override
+	public void deletePostalAddressByExternalReferenceCode(
+			String postalAddressExternalReferenceCode)
```

Comment:
is it called like this also in the "base"? I think should be only `externalReferenceCode`